### ISSUE TITLE
Grenade trigger issues

### ISSUE THREAD

**Issue by Angustmeta** [tgstation/tgstation#44479]
#### Some issues after extensive testing on local:

1. Infrared emitters can't be attached at all. 

2. Mousetrap doesn't work. With a mousetrap attached to the grenade it just stays a normal grenade (5 second timer). Doesn't matter if the mousetrap was armed when attached or not. 
 
~~3. Using health sensor (off) on wire will have the grenade prime normally (5 second timer) instead of toggling the health sensor's on/off state. Attaching a health sensor (on) will have it work, but it will still prime like a normal grenade of used in hand. I think using it in hand should toggle it on/off.~~ Intented. 

~~4. Using wirecutters to disarm primed grenades no longer works.~~ It ever working was apparently a bug. 

5. Disassembling an unsecured chemical grenade with a wrench will not make any attached trigger drop. Instead the trigger is still attached to the non-wired grenade casing. A QoL change would be to make them drop, so you can easily mass dissassemble grenades to use their triggers for something else (such as large grenades). 

~~6. We can probably remove most igniters from chemistry, since they're no longer used for triggers. 2 per chem lab for heating stuff would be enough.~~ Just bloats this issue. 
 
7. You can't remove a trigger from an unsecured grenade without using a multitool or another trigger on it first to open the hacking window. But using a multitool will also open the extra "please enter a new detonation time" window due to @nemvar 's change, which is a bit annoying. I suggest making it let you activate it in hand instead of having to use a multitool. 

I really wish I didn't have to spend 2.5+ hours testing, when I just wanted to do some quick wiki editing. 

#44258

@Tlaltecuhtli

**Comment by Tlaltecuhtli** [comment]
1) infrared dont work on wires on their own
2) fixed
3) fixed
4) seems unintended
5)fixed
6)if someone does some map changes to all 6 maps maybe remind him
7) fixed

**Comment by ExcessiveUseOfCobblestone** [comment]
in regards to 4,

> I didn't even know that existed tbh, let's try to retain that functionality.

**Comment by Tlaltecuhtli** [comment]
https://github.com/tgstation/tgstation/blob/88588de227a0abcc70540e6fe402b3bf81c054b5/code/game/objects/items/grenades/chem_grenade.dm#L102
wasnt touched by my pr and yeah it clearly says !active

**Comment by Angustmeta** [comment]
True, it seems intended for wirecutters to not work when it's active. 

On 22:nd april 2018 I updated the grenade wiki page with this changelog:

> Corrected tool used to disarm a grenade. Tested both on a ticking timed grenade and a non-activated but secured grenade. Screwdriver just toggles timer between 5 and 3 seconds on standard (wire) grenades. Also fixed small typos.

It was from a testing session over a year ago, where wirecutters did work to disarm live nades. Looks like that was a bug. 

From checking git blame, I assume it was fixed by #40414 or something. 

I liked that functionality though. Never managed to react fast enough to use it. I'll update issue.
